localization/replace/english/culture/traditions/wc_cultural_traditions_l_english.yml

```diff
@@ -0,0 +1,85 @@
+﻿l_english:
+
+ ##############################
+ # Adapted Vanilla Traditions #
+ ##############################
+
+ tradition_lords_of_the_elephant_name:0 "Lords of the Behemoths"
+ tradition_lords_of_the_elephant_desc:0 "It is said that every new Exarch is granted a great Elekk, whose will and loyalty serve as constant reminders of an Exarch's duty. When peace is broken, our Exarchs ride forth in a righteous fury, routing any who would harm the weak and innocent."
+ culture_parameter_elephant_bonuses_for_rank_4_education:0 "[rulers|E] with the [GetTrait('education_diplomacy_4').GetName( GetNullCharacter )], [GetTrait('education_stewardship_4').GetName( GetNullCharacter )], [GetTrait('education_learning_4').GetName( GetNullCharacter )] or [GetTrait('education_martial_4').GetName( GetNullCharacter )] [traits|E] gain significant [elephant_cavalry|E] bonuses"
+ tradition_lords_of_the_elephant_regional_desc:0 "This culture venerates different gargantuan beasts, likening them to royalty. They are masterful at capturing them, training them, and using them for devastating effect in war."
+
+ tradition_himalayan_settlers_name:0 "Kun Lai Pathfinders"
+ tradition_himalayan_settlers_desc:0 "Grummles have lived around Mount Neverest and navigated the Burlap Trail for as long as lorewalkers keep records. In times of strife, the winding wends and craggy peaks of Kun-Lai make good shelter and even better outposts for guerilla attacks that melt away into the wind and snow."
+ 
+ tradition_mountaineer_ruralism_name:0 "Hill Lodges"
+ tradition_mountaineer_ruralism_desc:0 "From the rugged highlands to the verdant valleys below, the trappers of the Grizzly Hills track across the varied geography with practiced ease and have established settlements in areas otherwise thought unsuitable to habitation."
```

Comment:
```suggestion
 tradition_mountaineer_ruralism_desc:0 "From the rugged highlands to the verdant valleys below, the trappers of the Grizzly Hills track across the varied geography with practiced ease and have established settlements in areas otherwise thought unsuitable for habitation."
```